### PR TITLE
[15.0][IMP] survey_skip_start: simpler implementation

### PR DESCRIPTION
**Pull request by chienandalu**
Simplifies the implementation and bring compatibility with any survey layout.

cc @Tecnativa TT47278

please review @pedrobaeza @victoralmau

### CODE REVIEW

File: survey_skip_start/models/survey_survey.py [hunk 1]
```diff
@@ -12,13 +11,7 @@ class SurveySurvey(models.Model):
...
-        """For the moment, skipping the start screen is only compatible with single
-        page layout surveys"""
-        if self.filtered(lambda x: x.skip_start and x.questions_layout != "one_page"):
-            raise UserError(
-                _(
-                    "The skip start screen option is only compatible with on page layouts"
-                )
-            )
+    def _create_answer(self, *args, **additional_vals):
+        inputs = super()._create_answer(*args, **additional_vals)
```
Comment (pedrobaeza) [inline survey_skip_start/models/survey_survey.py:17]: Please docstring about why are you overwriting this method.

**Review by pedrobaeza — COMMENTED** [review]
As this is hacking current process, you should document it properly for future references, and make sure the test covers the flow in case this fails.

File: survey_skip_start/models/survey_survey.py (hunk 1 above)
Comment (chienandalu) [reply survey_skip_start/models/survey_survey.py:17]: done :+1:

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**